norse/torch/functional/stdp.py

```diff
@@ -1,400 +1,276 @@
-import torch
+# Includes
+from typing import Tuple
 
-from typing import Tuple, NamedTuple, Union
+import torch
+from torch import clamp, relu, pow as exponentiate
```

Comment:
This import is confusing to me. I wouldn't import these separately, but just use `torch.pow`, `torch.clamp` and `torch.nn.functional.relu` (this is the same as `torch.relu`, but the code actually lives in `torch.nn.functional`, so this is more informative)